tests/kernel/multipath_iscsi.pm

```diff
@@ -23,6 +23,11 @@ sub run {
 
     select_serial_terminal;
 
+    # Check connectivity to any external target out of internal multimachine network
+    ping_size_check('openqa.opensuse.org');
```

Comment:
This adds a single point of failure. In other cases we avoid this, e.g. workers are not immediately giving up on jobs when the web UI host is shortly not reachable. It would be nice to add some fallback hosts and only fail the test if none of them are reachable.

Reply:
That was forgotten experiment, which should be removed.